helm/prometheus-meta-operator/templates/configmap.yaml

```diff
@@ -28,8 +28,6 @@ data:
         {{- range .Values.Installation.V1.Monitoring.Prometheus.Bastions }}
         - {{ . }}
         {{- end }}
-        remoteWrite:
-          url: https://prometheus-us-central1.grafana.net/api/prom/push
```

Comment:
Instead of removing this can we please add a toggle flag, something like
```diff
        remoteWrite:	
          url: https://prometheus-us-central1.grafana.net/api/prom/push
+         enable: false
```

So when we need to enable this, its just a matter of toggling this flag and not having to find what was this url.

Reply:
How about if we comment out the URL instead of removing it?

Cause I do get what you mean, it would be much simpler to be able to toggle it, but adding another setting will translate to adding another CLI flag, and from UX perspective I'm not a fan of patterns like:

```
--remote-write-enable --remote-write-url http://foo
```

cause if I pass `--remote-write-url` it implies I want it enabled.

So I'd slightly prefer if we were toggling it by commenting out, but if you think a flag is better, I'll add it.

Reply:
ok, fine go with commenting it. I just didn't wanted to loose the value in the end.

Reply:
Done